docs/index.rst

```diff
@@ -69,6 +69,35 @@ Example:
    to this mount.
 
 
+CDN
+~~~
+
+Crane can also be configured to support some advanced options when content resides
+on a Content Delivery Network. The URL match and replace parameters support rewriting
+the base paths of repos to a new location on the CDN.
```

Comment:
notion of `base path` might not be understood as clearly as we think.
What about re-phrasing this to 'URL where content is served' or similar?

for reference https://docs.pulpproject.org/plugins/pulp_docker/tech-reference/distributor.html#v4-redirect-file